### DISCUSSION TITLE
朱楚文：眼見不為憑！當AI詐騙連視訊都能仿造時，我們該如何分辨真假？

### DISCUSSION THREAD

**Discussion by Patsy17** [1101-GS4719-C/Python-Community-GS4719-C#1863] [General]
以前得詐騙可能都是用電話的方式來進行，現在隨著科技的發達，
詐騙集團開始會使用AI的人臉換臉進行視訊，讓受害者降低防備，
文章中提及李開復先生在AI 2041這本書中，提到未來人人出門都會帶面具，因為當不法分子可以擷取到你的面容，就可以生成各種虛擬詐騙，保護臉會成為我們的日常！

用想的就覺得好恐怖，我們應該要對所接受到的資訊更加留意、把關，才不會淪為詐騙的受害者

資料來源：https://www.thenewslens.com/article/197012

**Comment by FakeWood** [comment]
我雖然我沒看過這本書，但我覺得未來為了不被壞人擷取面容而人人帶面具出門的情況不太會發生，因為現在大家，至少年輕人，至少部分有sence的人，就已經懂得眼見不一定為憑，不會只相信影片中的內容了

**Comment by yubo0oo** [comment]
我覺得，詐騙的手段隨著科技的進步也在不斷升級，這真的是一個讓人非常擔心的趨勢。以前的詐騙多是通過電話進行，但現在詐騙集團已經開始使用AI技術的人臉換臉來進行視訊，這讓受害者更容易放鬆警惕，掉入陷阱。

文章中提到李開復先生在《AI 2041》這本書中提到，未來人人出門都會帶面具，因為不法分子一旦擷取到你的面容，就可以生成各種虛擬詐騙。想像一下，保護自己的臉竟然會成為我們的日常，這真的讓人覺得非常恐怖。

我覺得，我們應該對所接受到的資訊更加留意和把關，才不會淪為詐騙的受害者。隨著科技的不斷進步，防範意識也必須同步提升，才能更好地保護自己和家人。你們對這個問題有什麼看法呢？有什麼防範詐騙的好方法嗎？歡迎大家一起討論和分享！
